### PR TITLE
Fixed AAPM Restrictions from AEM Permission console

### PR DESCRIPTION
**Pull request by vivekmalviya0104**

### CODE REVIEW

File: core/src/main/java/com/valtech/aapm/restrictions/HasPropertyValuesPattern.java [hunk 1]
```diff
@@ -45,32 +45,31 @@ public class HasPropertyValuesPattern implements RestrictionPattern {
     private final String propertyType;
     private static final String DENY = "deny";
 
-    List<String> operatorList = List.of("==", ">=", "<=","<",">");
-    private String operator = "==";
+    List<String> operators = List.of("DOUBLE_EQUALS", "GREATER_THAN_EQUALS", "LESS_THAN_EQUALS", "GREATER_THEN", "LESS_THEN");
```
Comment (gruberrolandvaltech) [inline core/src/main/java/com/valtech/aapm/restrictions/HasPropertyValuesPattern.java:48]: I would replace "DOUBLE_EQUALS" with "EQUALS" as this is more related to the meaning of the comparison.

File: core/src/test/java/com/valtech/aapm/restrictions/PropertyValueRestrictionProviderTest.java [hunk 2]
```diff
@@ -74,20 +70,19 @@ public void getPattern_nominal_case() {
...
 
     @Test
-    public void getPattern_returns_empty_pattern_restriction_when_special_property_is_a_multiple_one() {
+    void getPattern_returns_empty_pattern_restriction_when_special_property_is_a_multiple_one() {
         Type<Iterable<String>> type = Type.STRINGS;
         String oakPath = "/my/path";
         String name = "hasPropertyValues";
-        String value1 = "deny#string$cq:tags==properties:orientation/portrait";
-        String value2 = "allow#string$cq:tags==properties:orientation/landscape";
+        String value1 = "deny#string$cq:tagsDOUBLE_EQUALSproperties:orientation/portrait";
```
Comment (gruberrolandvaltech) [inline core/src/test/java/com/valtech/aapm/restrictions/PropertyValueRestrictionProviderTest.java:84]: The readability of "deny#string$cq:tagsDOUBLE_EQUALSproperties:orientation/portrait" is not good.
How about adding a space before and after the operator?
Like "deny#string$cq:tags DOUBLE_EQUALS properties:orientation/portrait"

**Review by gruberrolandvaltech — CHANGES_REQUESTED** [review]
See comments

File: core/src/main/java/com/valtech/aapm/restrictions/HasPropertyValuesPattern.java (hunk 1 above)
Comment (vivekmalviya0104) [reply core/src/main/java/com/valtech/aapm/restrictions/HasPropertyValuesPattern.java]: Fixed

File: core/src/test/java/com/valtech/aapm/restrictions/PropertyValueRestrictionProviderTest.java (hunk 2 above)
Comment (vivekmalviya0104) [reply core/src/test/java/com/valtech/aapm/restrictions/PropertyValueRestrictionProviderTest.java]: having space in start and end is creating more problem therefore I added _

**Review by gruberrolandvaltech — APPROVED** [review]

**Pull request merged**